components/Navbar.vue

```diff
@@ -337,6 +337,10 @@ export default class NavbarMenu extends mixins(PrefixMixin) {
     }
   }
 
+  .koda-is-hidden-desktop {
+    background-color: $primary !important;
+  }
```

Comment:
this isn't behing used I think you can remove it

Reply:
done 👍